bin/btmon.py

```diff
@@ -1121,6 +1133,7 @@
 import traceback
 import urllib
 import urllib2
+from influxdb import InfluxDBClient
```

Comment:
yep, agreed! i did that in the follow-up commit. the first commit is from @chicago6061, then the rest are from me. I didn't want to commit things on his behalf, but happy to squash them if you'd prefer.